doc/conf/hosts.yaml

```diff
@@ -55,11 +55,15 @@ PROVISION:
     #user to provide ngcacher_proxy only in case when servers are behind company
     #firewall, else left blank.
     ngcacher_proxy: ""
-
   PXE:
     password: cable123
     server_ip: "10.197.121.21"
     user: ubuntu
+  POST:
+    mellanox:
```

Comment:
Are you sure mellanox is a good key to use here? Is this a construct that can be used for other reasons other than special handling for the Mellanox NIC?

Reply:
@spisarski Thanks for reviewing and your comments. I did think about this issue along the same line, so I share your concern.  However, I'd like to wait for another similar requirement to come in to better refactor it. I created #228 to track it.

Reply:
This could be made more generic, if you added a script link.  Then iso_url and other other parameters could just be in the script.

Could we automate the kernel version retrieval?  Looking in /target/lib or /lib to see what version is installed.  We could parse the parameter from the image package name, if it is set.